showcase-docs/getting-started.md

```diff
@@ -2,6 +2,151 @@
+        disabled: false
+```
+
+NOTE: If the `analytics-provider-segment` plugin is already present in your ConfigMap file, set the value of the `plugins.disabled` parameter to `true`.
+
+Set the value of the `dynamicPluginsConfigMapName` parameter to the name of the `ConfigMap` file in your `Backstage` custom resource:
+
+```yaml
+spec:
+  application:
+    dynamicPluginsConfigMapName: dynamic-plugins-rhdh
+```
+
+### Customizing Telemetry Destination
+
+The `analytics-provider-segment` plugin is configured to send data to Red Hat by default. To change the destination that receives telemetry data, set the value of the `SEGMENT_WRITE_KEY` environment variable in your Helm configuration file as shown in the following examples.
+
+#### Example using Helm Chart
+
+```yaml
+upstream:
+  backstage:
+    extraEnvVars:
+      - name: SEGMENT_WRITE_KEY
+        value: <segment_key>
+```
+
+#### Example using RHDH Operator
+
+```yaml
+extraEnvs:
+  envs:
+    - name: SEGMENT_WRITE_KEY
+      value: <segment_key>
+```
+
+If you wish to subsequently disable telemetry data collection, use one of the following methods described below.
+
+### Disable Telemetry
+
+To turn off the telemetry feature, you must disable the `analytics-provider-segment` plugin either using the Helm Chart or the RHDH Operator.
+
+#### Using Helm Chart
+
+Add the following code in your Helm configuration file:
+
+```yaml
+global:
+  dynamic:
+    plugins:
+      - package: './dynamic-plugins/dist/janus-idp-backstage-plugin-analytics-provider-segment'
+        disabled: true
+```
+
+#### Using RHDH Operator
+
+When using RHDH Operator, you must modify the `ConfigMap` file created for dynamic plugin configuration. You specify the name of this `ConfigMap` file in the `dynamicPluginsConfigMapName` field of your `Backstage` custom resource. Usually, the `ConfigMap` file is named as `dynamic-plugins-rhdh`.
+Add the following code in your `ConfigMap` file:
+
+```yaml
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: dynamic-plugins-rhdh
+data:
+  dynamic-plugins.yaml: |
+    includes:
+      - dynamic-plugins.default.yaml
+    plugins:
+      - package: './dynamic-plugins/dist/janus-idp-backstage-plugin-analytics-provider-segment'
+        disabled: true
+```
+
+### Disable Telemetry for Local Development
+
+By default, the `analytics-provider-segment` plugin is disabled when you run your application locally without using the `dynamic-plugins.default.yaml` file.
+However, if you run your application using the `dynamic-plugins.default.yaml` file, you can disable the `analytics-provider-segment` plugin as shown in the following example:
+
+```yaml
+dynamicPlugins:
+  plugins:
+    - package: './dynamic-plugins/dist/janus-idp-backstage-plugin-analytics-provider-segment'
+      disabled: true
+```
+
+Delete the `dynamic-plugins-root/janus-idp-backstage-plugin-analytics-provider-segment` plugin directory.
```

Comment:
is this a second step, or an alternative step to the above configuration change?

Reply:
Second step. Do you think I should use numbered list to make it clearer?

Reply:
updarted